src/assets/styles/_typography.scss

```diff
@@ -10,3 +10,22 @@
     font-size: 1.429rem;
   }
 }
+// Title with font KobzarKS should be in the div tag
+
+.kobzar-title {
```

Comment:
I would not tie title with font name, because font can be changed. 
But t's fine for now, don't need to be change, just FYI

Reply:
Yes, I agree. In this case, have a problem with the specific font or if be more precise with font line height. So for this reason I added a specific configuration for this font.